src/lib/useOnScreen.ts

```diff
@@ -13,7 +13,7 @@ export default function useOnScreen(
     const observer = new IntersectionObserver(([entry]) =>
       setIsOnScreen(entry.isIntersecting)
     );
-    if (ref.current) {
+    if (ref?.current) {
```

Comment:
was making tests fail

Reply:
Have they been failing again recently?? Or was this due to your changes here?

Reply:
this was due to my changes. Since `useOnScreen` was complaining about ref being undefined.